tools/operator/connectionprofile/connectionprofile.go

```diff
@@ -91,6 +100,221 @@ func (c ConnProfile) GenerateConnProfilePerOrg(orgName string) error {
 	return nil
 }
 
+func (c ConnProfile) orderers2caliper(ordererMap map[string]networkspec.Orderer) (map[string]networkspec.CaliperOrderer, error) {
+	caliperOrderers := make(map[string]networkspec.CaliperOrderer)
+	for o := range ordererMap {
+		orderer := ordererMap[o]
+		caliperOrderer := networkspec.CaliperOrderer{URL: orderer.URL, GrpcOptions: orderer.GrpcOptions, TLSCACerts: orderer.TLSCACerts}
+		caliperOrderers[o] = caliperOrderer
+	}
+
+	return caliperOrderers, nil
+}
+
+func (c ConnProfile) peers2caliper(peerMap map[string]networkspec.Peer) (map[string]networkspec.CaliperPeer, error) {
+	caliperPeers := make(map[string]networkspec.CaliperPeer)
+	for p := range peerMap {
+		peer := peerMap[p]
+		caliperPeer := networkspec.CaliperPeer{URL: peer.URL, GrpcOptions: peer.GrpcOptions, TLSCACerts: peer.TLSCACerts}
+		caliperPeers[p] = caliperPeer
+	}
+
+	return caliperPeers, nil
+}
+
+func (c ConnProfile) ca2caliper(caMAP map[string]networkspec.CertificateAuthority) (map[string]networkspec.CaliperCertificateAuthority, error) {
+	caliperCAs := make(map[string]networkspec.CaliperCertificateAuthority)
+	for c := range caMAP {
+		ca := caMAP[c]
+		caliperCA := networkspec.CaliperCertificateAuthority{URL: ca.URL, CAName: ca.CAName, TLSCACerts: ca.TLSCACerts, HTTPOptions: ca.HTTPOptions}
+		caReg := networkspec.CARegistrar{EnrollID: ca.Registrar.EnrollID, EnrollSecret: ca.Registrar.EnrollSecret}
+		caliperCA.Registrar = append(caliperCA.Registrar, caReg)
+		caliperCAs[c] = caliperCA
+	}
+
+	return caliperCAs, nil
+}
+
+func (c ConnProfile) orgs2caliper(orgMAP map[string]networkspec.Organization) (map[string]networkspec.CaliperOrganization, error) {
+	caliperOrgs := make(map[string]networkspec.CaliperOrganization)
+	for o := range orgMAP {
+		org := orgMAP[o]
+		caliperOrg := networkspec.CaliperOrganization{MSPID: org.MSPID, Peers: org.Peers, CertificateAuthorities: org.CertificateAuthorities, AdminPrivateKey: org.AdminPrivateKey, SignedCert: org.SignedCert}
+		caliperOrgs[o] = caliperOrg
+	}
+
+	return caliperOrgs, nil
+}
+
+func (c ConnProfile) connprofile2caliper(connProfile ConnProfile) (CaliperConnProfile, error) {
+	caliperConnProfile := CaliperConnProfile{}
+
+	var err error
+	caliperConnProfile.Orderers, err = c.orderers2caliper(connProfile.Orderers)
+	if err != nil {
+		return caliperConnProfile, err
+	}
+
+	caliperConnProfile.Peers, err = c.peers2caliper(connProfile.Peers)
+	if err != nil {
+		return caliperConnProfile, err
+	}
+
+	caliperConnProfile.CA, err = c.ca2caliper(connProfile.CA)
+	if err != nil {
+		return caliperConnProfile, err
+	}
+
+	caliperConnProfile.Organizations, err = c.orgs2caliper(connProfile.Organizations)
+	if err != nil {
+		return caliperConnProfile, err
+	}
+
+	caliperConnProfile.Config = connProfile.Config
+
+	return caliperConnProfile, nil
+}
+
+func (c ConnProfile) clientCaliper(orgName string) (map[string]networkspec.CaliperClient, error) {
+	clients := make(map[string]networkspec.CaliperClient)
```

Comment:
Move the `clients` declaration down next to where it is first used

Reply:
ok